perses/rjmc/geometry.py

```diff
@@ -93,6 +94,7 @@ def __init__(self, metadata=None, use_sterics=False, verbose=True):
         self._position_set_time = 0.0
         self.verbose = verbose
         self.use_sterics = use_sterics
+        self._logger = logging.getLogger("geometry")
```

Comment:
It is also common to instantiate a single `logger` at the module level. This works too though!

Reply:
I gave it a run and it seems to respond as I expected from the calling module, so I think we're good there. INFO isn't too bad here--just the DEBUG stuff is crazy.